vspec2csv.py

```diff
@@ -48,11 +48,18 @@ def usage():
 """)
     sys.exit(255)
 
-
+#Format a data or header line according to the CSV standard (IETF RFC 4180)
+def format_csv_line(*csv_fields):
+    formatted_csv_line = ""
+    for csv_field in csv_fields:
+        formatted_csv_line = formatted_csv_line + '"' + str(csv_field).replace('"', '""') + '",'
+    return formatted_csv_line[:-1] + '\n'
+
+#Write the header line
 def print_csv_header(file):
-    file.write("Signal,Type,DataType,Deprecated,Complex,Unit,Min,Max,Desc,Enum,Id,Instance\n")
-
+    file.write(format_csv_line("Signal","Type","DataType","Deprecated","Complex","Unit","Min","Max","Desc","Enum","Id","Instance"))
```

Comment:
Right now, there are no issues with the header fields, but better to call the new function consistently throughout the code. Also makes it future proof too e.g. in case the field names are ever changed.